gbfs-validator/versions/partials/v3.0-RC2/vehicle_status/required_vehicle_type_id.js

```diff
@@ -0,0 +1,64 @@
+module.exports = ({ vehicleTypes }) => {
+  const res = {
```

Comment:
Slightly better than "r", but if it means "result", can we just name it "result"? Or I would even argue that "partial" is a good variable name here.

Reply:
Thank you for the suggestion! I renamed the variable to `partial`.